rpc/rpc-wrappers.go

```diff
@@ -1373,6 +1415,12 @@ func (cac StorageAuthorityClient) FinalizeAuthorization(authz core.Authorization
 	return
 }
 
+// RevokeAuthorization sends a request to revoke a pending or finalized authorization
+func (cac StorageAuthorityClient) RevokeAuthorization(id string) (err error) {
+	_, err = cac.rpc.DispatchSync(MethodRevokeAuthorization, []byte(id))
```

Comment:
Please make this a json hash. We'll have a much harder time dealing with the RPC layer when there are all of these special cases.